python/lsst/pipe/tasks/selectImages.py

```diff
@@ -524,8 +524,17 @@ def run(self, visitSummaries, skyMap, dataId):
             output = sortedVisits
         else:
             output = sortedVisits[:self.config.nVisitsMax]
-        self.log.info("%d images selected with FWHM range of %f--%f arcseconds",
-                      len(output), fwhmSizes[visits.index(output[0])], fwhmSizes[visits.index(output[-1])])
+
+        if len(output) == 0:
+            self.log.info("All images rejected in BestSeeingSelectVisitsTask.")
+            raise pipeBase.NoWorkFound(f'No good images found.')
```

Comment:
Don't think you need the `f` if nothing is getting evaluated (and I wouldn't mind seeing the dataId added 😉).